app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileShowMe.kt

```diff
@@ -14,8 +14,10 @@ import kotlinx.serialization.json.Json
 
 private const val NONE_CHECKED = -1
 
+/**
+ * Activity that allow the user to specify which gender he wants to see.
```

Comment:
Maybe we could use "they" as something neutral when referring to a user

Reply:
Sure !